lib/minfraud/transaction.rb

```diff
@@ -85,7 +85,7 @@ def timeout=(timeout)
     # Ensures the required attributes are present
     # @return [Boolean]
     def has_required_attributes?
-      ip and city and state and postal and country and txn_id
+      !!ip
```

Comment:
Leave `txn_id` as required. Without it, we cannot track down where the minfraud request came from.